mobile/src/bundles/common/components/form-checkbox/form-checkbox.tsx

```diff
@@ -0,0 +1,54 @@
+import CheckBox, { type CheckBoxProps } from '@react-native-community/checkbox';
+import React from 'react';
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { useFormController } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+type Properties<T extends FieldValues> = CheckBoxProps & {
+    label?: string;
+    name: FieldPath<T>;
+    control: Control<T, null>;
+    containerStyle?: StyleProp<ViewStyle>;
+};
+
+const FormCheckbox = <T extends FieldValues>({
```

Comment:
We already have checkbox group component

Reply:
I understood that you need it for a single checkbox, let's ask the product owner if we can use a switch component for that. It's more mobile friendly